docs/develop/update/firewall.rst

```diff
@@ -65,3 +69,116 @@ When making changes to firewall settings or port assignments, you might want to:
    .. code-block:: bash
 
       netstat -tupln
+
+Docker Servers
+==============
+
+Docker manipulates IPTables to route network traffic to and from containers this interferes with our firewall management script. To prevent errors saltstack and the firewall script identify docker and block the script from running. To work around this on servers hosting Docker we need to implement an alternative firewall solution that is configured external to the instance.
+
+Hetzner (Hardware Servers)
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Hetzner provide a simple stateless firewall with each server. "Stateless" means that the firewall does not track connections, it simply monitors all inbound and outbound traffic by IP and ports in that moment.
```

Comment:
```suggestion
Hetzner provide a free `stateless firewall <https://docs.hetzner.com/robot/dedicated-server/firewall/>`__ for each dedicated server. "Stateless" means that the firewall does not store information about connections across time, which is required for use cases like port knocking.
```

Reply:
The "stateless" firewall caught us out on Registry because it was blocking the response from outbound requests, for instance running `curl` to get a web page. 
I'm not sure how to phrase it but I think this would be a better example than port knocking because users are much more likely to run into this. (or perhaps to add as well as port knocking)

Reply:
Maybe "which is required for HTTP sessions and port knocking, for example."